### DISCUSSION TITLE
Problem with page made by a repository

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22869] [Other Feature Feedback, Questions, & Ideas]
<p>I made a blog with Github Pages and Jekyll. For my Persian blog, i made a repository named farsi and converted it to a page so it would be available at <a href="https://alireza.one/farsi" rel="nofollow noopener">https://alireza.one/farsi</a>. Now sometmes when i open my blog by my iphone, page doesn’t open and it appears like a file and it wants to open in Telegram messenger. </p>
<p>Can somebody tell me what should i do?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/589e980f2a420f1ca6e24be4685f5c41875b5e8e.jpeg" data-download-href="/uploads/short-url/cDXNOpFNDVr8nHWOlvunSLc4BGe.jpg?dl=1" title="F2BA8715-9271-488D-AB28-8800CE31E5AE.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10137/181073604-fd244201-dcb3-45af-ad01-c7b4d51579b1.jpg" alt="F2BA8715-9271-488D-AB28-8800CE31E5AE.png" data-base62-sha1="cDXNOpFNDVr8nHWOlvunSLc4BGe" width="281" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/589e980f2a420f1ca6e24be4685f5c41875b5e8e_2_281x500.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/589e980f2a420f1ca6e24be4685f5c41875b5e8e_2_421x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/589e980f2a420f1ca6e24be4685f5c41875b5e8e_2_562x1000.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/589e980f2a420f1ca6e24be4685f5c41875b5e8e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">F2BA8715-9271-488D-AB28-8800CE31E5AE.png</span><span class="informations">1242×2208 118 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by oycyc** [comment, marked as answer]
<p>It looks fine to me, from the computer and from my phone(I have an iPhone as well). Perhaps you refreshed it too quickly after you just changed the name of the repository? Sometimes, it takes a while to update. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/85d921d7e83bf0d39831fa8aa5c318ce120f014e.jpeg" data-download-href="/uploads/short-url/j64HQl3fj28E7Qu1y9ZWliV86J0.jpg?dl=1" title="image_6483441.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/35899715/181073606-85d8589c-3cc8-447e-915f-7b2e80db58a8.jpg" alt="image_6483441.JPG" data-base62-sha1="j64HQl3fj28E7Qu1y9ZWliV86J0" width="281" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/85d921d7e83bf0d39831fa8aa5c318ce120f014e_2_281x500.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/8/85d921d7e83bf0d39831fa8aa5c318ce120f014e_2_421x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/85d921d7e83bf0d39831fa8aa5c318ce120f014e_2_562x1000.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/85d921d7e83bf0d39831fa8aa5c318ce120f014e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image_6483441.JPG</span><span class="informations">640×1136 52.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ghost** [comment]
<p>No i didnt change the name. i build it about the week ago and i get this message from last night. It doesn’t happen all the time but it happens some times.</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/alirezahayati">@alirezahayati</a>,</p>
<p>I just took a look, and I’m actually now getting a 404 when trying to visit your website. Did you change anything since you posted this? You may need to reach out to <a href="https://github.com/contact" rel="nofollow noopener">private support</a> to get further help; they can take a look at your account in detail.</p>

**Comment by ghost** [comment]
<p>Yeah, i took it down.</p>
